### PR TITLE
Design Navbar

### LINKED ISSUE #20: Navigation Bar
Design and Add nav bar to web app

### PR DESCRIPTION
**Pull request by tonae-patterson**
Navbar to be situated in app header

### CODE REVIEW

File: Frontend/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
-  exports: [RouterModule]
-})
+      imports: [RouterModule.forRoot(routes)],
```
Comment (adamchlebek) [inline Frontend/src/app/app-routing.module.ts]: Remove whitespace changes

File: Frontend/src/styles.scss [hunk 2]
```diff
@@ -8,6 +8,7 @@
   @include nb-theme-global();
   @include nb-auth-global(); // append the install mixin inside of the nb-install
 };
+
```
Comment (adamchlebek) [inline Frontend/src/styles.scss]: Remove whitespace

**Review by adamchlebek — COMMENTED** [review]
Small whitespace changes

**Review by adamchlebek — COMMENTED** [review]
I noticed that if you click the name dropdown, you are unable to close it.

File: Frontend/src/app/app-routing.module.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { AuthGuard } from './core/guards/auth.guard';
 const routes: Routes = [
   {
     path: 'auth',
-    canActivate: [ AuthGuard ],
+    canActivate: [AuthGuard],
```
Comment (adamchlebek) [inline Frontend/src/app/app-routing.module.ts]: Unnecessary whitespace changes

**Review by adamchlebek — COMMENTED** [review]
Whitespace changes

**Review by jwaltt55 — APPROVED** [review]
